wire-ios/Wire-iOS/Sources/UserInterface/Conversation Options/CreateSecureGuestLink/CreateSecureConversationGuestLinkViewModel.swift

```diff
@@ -62,6 +76,31 @@ final class CreateSecureGuestLinkViewModel {
         return true
     }
 
+    func createSecuredGuestLinkIfValid(conversation: ZMConversation, passwordField: ValidatedTextField, confirmPasswordField: ValidatedTextField) {
+        guard validatePassword(for: passwordField, against: confirmPasswordField) else {
+            delegate?.viewModel(self, didFailToValidatePasswordWithReason: "Password validation failed.")
+            return
+        }
+
+        let password = passwordField.text ?? ""
```

Comment:
I suggest to unwrap the password rather than default to empty.

Reply:
Done! https://github.com/wireapp/wire-ios/commit/e03678e15010db6793ed438fc93973ddab433709